.env.example

```diff
@@ -10,6 +10,6 @@ WILLS_URL = "https://ewills-dev.vs.gov.bc.ca/"
 BCOL_URL = "https://d1.bconline.gov.bc.ca"
 ADDRESSCHANGE_URL = "https://www.addresschange.gov.bc.ca/"
 REGISTRY_ACCOUNT_CREATE_URL = "https://dev.bcregistry.ca/business/auth/choose-authentication-method"
-KEYCLOAK_CONFIG_PATH = "/local-keycloak-config-url/keycloak.json"
+KEYCLOAK_CONFIG_PATH = "keycloak-dev.json"
```

Comment:
This needs to be configured for dev/test/prod.

Reply:
Also to prevent caching issues, the web server config should never cache "index.html" ([example](https://github.com/bcgov/business-filings-ui/commit/ec1e52f782c752ccab37ddc68594ccbc9e93e4a8#diff-1f91f64bfa3fb0a1ece881887af0a2356b8c529a96fb91c1894745b2a1a009aa)).